src/sync.sh

```diff
@@ -18,6 +18,34 @@ cp -fvr theme/* /opt/keycloak/themes/theme/
 cp -fv *.jar /opt/keycloak/providers/
 cp -fv certs/* /opt/keycloak/conf/truststores
 
+if [ -d /opt/keycloak/theme-overrides ]; then
+    echo "Applying theme customizations"
+    if [ -f /opt/keycloak/theme-overrides/logo.svg ]; then
+      echo "Overriding logo.svg"
+      cp -fv /opt/keycloak/theme-overrides/logo.svg /opt/keycloak/themes/theme/login/resources/img/logo.svg
+      cp -fv /opt/keycloak/theme-overrides/logo.svg /opt/keycloak/themes/theme/login/resources/img/uds-logo.svg
+      cp -fv /opt/keycloak/theme-overrides/logo.svg /opt/keycloak/themes/theme/account/resources/public/logo.svg
+      cp -fv /opt/keycloak/theme-overrides/logo.svg /opt/keycloak/themes/theme/account/resources/public/uds-logo.svg
+    fi
+
+    if [ -f /opt/keycloak/theme-overrides/favicon.svg ]; then
+      echo "Overriding favicon.svg"
+      cp -fv /opt/keycloak/theme-overrides/favicon.svg /opt/keycloak/themes/theme/login/resources/img/favicon.svg
+    fi
+
+    if [ -f /opt/keycloak/theme-overrides/background.jpg ]; then
+      echo "Overriding background.jpg"
+      cp -fv /opt/keycloak/theme-overrides/background.jpg /opt/keycloak/themes/theme/login/resources/img/tech-bg.jpg
+      cp -fv /opt/keycloak/theme-overrides/background.jpg /opt/keycloak/themes/theme/account/resources/public/tech-bg.jpg
+    fi
+
+    if [ -f /opt/keycloak/theme-overrides/footer.png ]; then
+      echo "Overriding footer.png"
+      cp -fv /opt/keycloak/theme-overrides/footer.png /opt/keycloak/themes/theme/login/resources/img/full-du-logo.png
+      cp -fv /opt/keycloak/theme-overrides/footer.png /opt/keycloak/themes/theme/account/resources/public/full-du-logo.png
```

Comment:
Seeing this here - should we change the file reference to be more generic for the logo (`full-logo` instead of `full-du-logo`)? This isn't really something the end user would see unless they were looking at the page source so it might not matter. Similar thought on the `uds-logo` above.

Reply:
I was thinking about renaming the theme files but I'm a bit hesitant to do this (especially in this PR).

I know some Mission Heroes fork this repo and provide their own customizations. If we changed the file names without giving them a chance to mirror our changes in their repo, we might end up in a messy state.

I created https://github.com/defenseunicorns/uds-identity-config/issues/390 to find these inconsistencies and align them.

Reply:
Sounds good - I agree. Probably best to wait on or maybe not do.